[2025-03-17T04:32:12.860+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T04:32:12.909+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_etl.clean_data manual__2025-03-17T04:31:23.449468+00:00 [queued]>
[2025-03-17T04:32:12.941+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_etl.clean_data manual__2025-03-17T04:31:23.449468+00:00 [queued]>
[2025-03-17T04:32:12.948+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-03-17T04:32:13.031+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): clean_data> on 2025-03-17 04:31:23.449468+00:00
[2025-03-17T04:32:13.052+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'exchange_rate_etl', 'clean_data', 'manual__2025-03-17T04:31:23.449468+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_pipeline.py', '--cfg-path', '/tmp/tmp9_5tpz_h']
[2025-03-17T04:32:13.059+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask clean_data
[2025-03-17T04:32:13.061+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=928) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T04:32:13.069+0000] {standard_task_runner.py:72} INFO - Started process 946 to run task
[2025-03-17T04:32:13.255+0000] {task_command.py:467} INFO - Running <TaskInstance: exchange_rate_etl.clean_data manual__2025-03-17T04:31:23.449468+00:00 [running]> on host 8ae3f6e178cd
[2025-03-17T04:32:13.503+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='exchange_rate_etl' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T04:31:23.449468+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T04:31:23.449468+00:00'
[2025-03-17T04:32:13.505+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T04:32:13.610+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T04:32:13.636+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T04:32:13.637+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=exchange_rate_etl, task_id=clean_data, run_id=manual__2025-03-17T04:31:23.449468+00:00, execution_date=20250317T043123, start_date=20250317T043212, end_date=20250317T043213
[2025-03-17T04:32:13.771+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T04:32:13.835+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T04:32:13.855+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
